src/components/ProjectNavbar/ProjectNavbar.tsx

```diff
@@ -118,14 +118,14 @@ export default function Sidebar(props: any) {
                     {props.links.map((link: any, index: any) => (
                         <ListItem key={link.title}>
                             <ListItemButton component={Link} to={link.ref}>
-                                <Typography color = '#5B7E98' marginLeft='-25px'>
+                                <Typography color = '#5B7E98' marginLeft='0px'>
```

Comment:
Perhaps we don't need to set the marginLeft property in this element if it's going to be 0 🤔 

```suggestion
                                <Typography color = '#5B7E98'>
```